.github/workflows/test.yml

```diff
@@ -55,15 +55,15 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
-      - name: Install Go
-        uses: actions/setup-go@v3
+      - name: Download Go tip
+        uses: robinraju/release-downloader@efa4cd07bd0195e6cc65e9e30c251b49ce4d3e51 # v1.8
```

Comment:
`gh cli` is included in actions. Could we use [gh download](https://cli.github.com/manual/gh_run_download) instead?

Reply:
Great idea, yeah! 🎉 🚀